website/highlights/2020-06-11-introduce-wasm-plugins.md

```diff
@@ -0,0 +1,85 @@
+---
+last_modified_on: "2020-07-13"
+$schema: "/.meta/.schemas/highlights.json"
+title: "Experimental WASM Plugins (Dev builds only)"
+description: "Vector can be optionally built with an integrated WASM runtime"
+author_github: "https://github.com/hoverbear"
+hide_on_release_notes: false
+pr_numbers: [2006]
+release: "0.10.0"
+tags: ["type: new feature", "domain: transforms"]
+---
+
+import Alert from '@site/src/components/Alert';
+
+If you're feeling particularly brave, you can now test drive the WASM support in Vector!
+
+First, some big caveats:
+
+* You'll need to **build your own version of Vector**.
+  * Our runtime is a bit larger than we want, so we're working on slimming it down before we enable it by default!
+* Only **transforms** are supported right now.
+* The **API is unstable**.
+  * We have plans to refactor absolutely all of it and make it much better. We intend to work with any new plugins that emerge to make sure their plans are achievable. Don't be a stranger. 😁
+* Our WASM plugins don't yet have robust **watchdog timers** or **stream integration**. These are coming!
```

Comment:
Given these caveats, I'm wondering if this is worth a highlight? It would be nice to announce this with a guide that we can simply link to.